console_backend/src/types.rs

```diff
@@ -72,8 +75,13 @@ impl MsgSender {
     const LOCK_FAILURE: &'static str = "failed to aquire sender lock";
 
     pub fn new<W: Write + Send + 'static>(writer: W) -> Self {
+        let file = File::create("ntrip_output").ok();
+        if file.is_none() {
+            error!("could not create file!");
+        }
```

Comment:
This is for the purpose of debugging at the moment it seems
- Sending RTCM not working for Piksi Triple device.
- Works fine with strsvr just not via ntrip tab.
- Sending RTCM converted to SBP data works on other devices.

Reply:
File can't be created in `.app` 🤔